File: joiner-core/src/main/java/cz/encircled/joiner/exception/JoinerException.java
package cz.encircled.joiner.exception;

/**
 * Basic class for all Joiner runtime exceptions
 *
 * @author Kisel on 25.01.2016.
 */
public class JoinerException extends RuntimeException {

    public JoinerException(String message) {
        super(message);
    }

    public JoinerException(String message, Exception exception) {
        super(message, exception);
    }
}

File: joiner-core/src/main/java/cz/encircled/joiner/exception/JoinerException.kt
package cz.encircled.joiner.exception

/**
 * Basic class for all Joiner runtime exceptions

 * @author Kisel on 25.01.2016.
 */
open class JoinerException : RuntimeException {

    constructor(message: String) : super(message) {
    }

    constructor(message: String, exception: Exception) : super(message, exception) {
    }
}
